add cybergear macro25 keyboard (#12518)
qmk/qmk_firmware: keyboards/cybergear/macro25/readme.md

> # CyberGeear Macro25

Reviewer: ```suggestion
# CyberGear Macro25
```

> ![image](https://github.com/CyberGear/mk-macro-01/raw/master/images/result-preview.jpg)

Reviewer: ```suggestion
![](https://i.imgur.com/Jk2xGMvh.jpg)
```

> Macro keyboard meade using arduino pro micro, with VIA support

Reviewer 2: ```suggestion
Macro keyboard made using Arduino Pro Micro, with VIA support.
```

> * Keyboard Maintainer: [CyberGeear](https://github.com/CyberGear)
> * Hardware Supported: arduino pro micro
> * Hardware Availability: [repo](https://github.com/CyberGear/mk-macro-01)

Reviewer 2: ```suggestion
* Keyboard Maintainer: [CyberGear](https://github.com/CyberGear)
* Hardware Supported: Arduino Pro Micro
* Hardware Availability: [GitHub](https://github.com/CyberGear/mk-macro-01)
```

> compiling (after setting up your qmk build environment):
> 
>     qmk compile -kb cybergear/macro25 -km CyberGear

Reviewer: ```suggestion
Make example for this keyboard (after setting up your build environment):

    make cybergear/macro25:default
```

> See the [Setting Up Your QMK Environment](https://docs.qmk.fm/#/newbs_getting_started)

Reviewer: ```suggestion
See the [build environment setup](https://docs.qmk.fm/#/getting_started_build_tools) and the [make instructions](https://docs.qmk.fm/#/getting_started_make_guide) for more information. Brand new to QMK? Start with our [Complete Newbs Guide](https://docs.qmk.fm/#/newbs).
```
